scripts/user-login-migration-rollback.js

```diff
@@ -0,0 +1,118 @@
+const mongoose = require('mongoose');
+
+const { Schema } = mongoose;
+const { program } = require('commander');
+const { ObjectId } = require('@mikro-orm/mongodb');
+
+program
+	.requiredOption('-u, --url <value>', '(Required) URL of the MongoDB instance')
+	.requiredOption('-s, --schoolId <value>', 'The schoolId of the migrated school, that should be roll-backed.');
+program.parse();
+
+const options = program.opts();
+const mongodbUrl = options.url;
+const { schoolId } = options;
+
+const close = async () => mongoose.connection.close();
+
+const connect = async () => {
+	const mongooseOptions = {
+		useNewUrlParser: true,
+		useUnifiedTopology: true,
+	};
+
+	return mongoose.connect(mongodbUrl, mongooseOptions);
+};
+
+const User = mongoose.model(
```

Comment:
Whats about the whole school?

Reply:
the school should not be rolled back, because this script should heklp with mishaps of the migration wizard, not return the school to its pre-migrated state.